spec/features/scientists/show_spec.rb

```diff
@@ -0,0 +1,89 @@
+    @experiment2 = Experiment.create!(name: "Fire Flowers", objective: "To study and understand the biological transformation caused by ingesting a fire flower.", num_months: 333)
+    @experiment3 = Experiment.create!(name: "Breathing Fire", objective: "To amplify the destructive power of Bowser's fire breath.", num_months: 666)
+    @experiment4 = Experiment.create!(name: "Invincibility Stars", objective: "To study and harness the omnipotent power of the invincibility star.", num_months: 777)
+
+    @se_1 = ScientistExperiment.create!(scientist_id: @scientist1.id, experiment_id: @experiment1.id)
+    @se_2 = ScientistExperiment.create!(scientist_id: @scientist1.id, experiment_id: @experiment2.id)
+    @se_3 = ScientistExperiment.create!(scientist_id: @scientist1.id, experiment_id: @experiment4.id)
+    @se_4 = ScientistExperiment.create!(scientist_id: @scientist5.id, experiment_id: @experiment1.id)
+    @se_5 = ScientistExperiment.create!(scientist_id: @scientist5.id, experiment_id: @experiment3.id)
+    @se_6 = ScientistExperiment.create!(scientist_id: @scientist5.id, experiment_id: @experiment4.id)
+  end
+
+  context "As a visitor" do
+    describe "when I visit a scientist's show page," do
+      it "I see all of that scientist's information (name, specialty, alma mater)," do
+        visit scientist_path(@scientist1)
+
+        within("#scientist_info")
+
+        expect(page).to have_content("Name: #{@scientist1.name}")
+        expect(page).to have_content("Specialty: #{@scientist1.specialty}")
+        expect(page).to have_content("Alma Mater: #{@scientist1.university}")
+
+        expect(page).to_not have_content("Name: #{@scientist5.name}")
+        expect(page).to_not have_content("specialty: #{@scientist5.specialty}")
+        expect(page).to_not have_content("Alma Mater: #{@scientist5.university}")
+      end
+
+      it "I see the name of the lab where this scientist works," do
+        visit scientist_path(@scientist1)
+
+        within("#scientist_info")
+
+        expect(page).to have_content("Lab: #{@scientist1.lab.name}")
+        expect(page).to_not have_content("Lab: #{@scientist5.lab.name}")
+      end
+
+      it "and I see the names of all of the experiments this scientist is running." do
+        visit scientist_path(@scientist1)
+
+        within("#scientist_experiments")
+
+        expect(page).to have_content("#{@experiment1.name}")
+        expect(page).to have_content("#{@experiment2.name}")
+        expect(page).to have_content("#{@experiment4.name}")
+
+        expect(page).to_not have_content("#{@experiment3.name}")
+
+      end
+
+      it "Then next to each experiment's name, I see a button to remove that experiment from that scientist's work load" do
+        visit scientist_path(@scientist1)
+        
+        expect(page).to have_button("Remove")
+      end
```

Comment:
This test says that a button should show up next to every experiment, but your assertions just checks that the page has a button with Remove on it. This test doesn't prove the feature request in the US.